packages/editor/src/lib/editor/Editor.ts

```diff
@@ -981,6 +994,23 @@ export class Editor extends EventEmitter<TLEventMap> {
 		return this
 	}
 
+	/**
+	 * Squash the history to the given mark id.
```

Comment:
could you expand on what this means? I actually don't know what history squashing does 😅

Reply:
essentially it removes any marks between the top of the undo stack and the given mark id. So that then next time you hit undo you'll stop at the given mark id instead of some intermediate mark.

this is currently only used when completing a cropping operation, that I'm aware of.

Reply:
Ah makes sense! We should include this description in the comment